_sips/minutes/2017-10-24-sip-minutes.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: sips
+title: SIP Meeting Minutes - 24th October 2017
+
+partof: minutes
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+|Topic|Reviewers| Accepted/Rejected |
+| --- | --- | --- |
+| [SIP-34: Right-Associative By-Name Operators](http://docs.scala-lang.org/sips/right-associative-by-name-operators.html) | Adriaan Moors | Accepted |
+| [SIP-35: Opaque types](http://docs.scala-lang.org/sips/opaque-types.html) | Sébastien Doeraene | Pending |
+| [SIP-33: Match infix and prefix types to meet expression rules](http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html)| Josh Suereth | Pending |
```

Comment:
@jsuereth please give the feedback to the author (see below what was discussed, if you didn't already take a look (o: